test/acceptance/framework/k8s/kubectl.go

```diff
@@ -1,10 +1,11 @@
-package helpers
+package k8s
 
 import (
 	"github.com/gruntwork-io/terratest/modules/k8s"
-	"github.com/gruntwork-io/terratest/modules/logger"
+	terratestLogger "github.com/gruntwork-io/terratest/modules/logger"
 	"github.com/gruntwork-io/terratest/modules/shell"
 	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/hashicorp/consul-helm/test/acceptance/framework/logger"
 	"github.com/stretchr/testify/require"
 )
 
```

Comment:
This comment is regarding L12-15.

I know you made a contribution to terratest to pass in a custom logger for the tunnel struct, and I was wondering if we'd be able to leverage changes from their larger logging change https://github.com/gruntwork-io/terratest/pull/510 to not have to reimplement these methods anymore?

Reply:
Good question! The reason we're re-implementing some of them is so that we can set our own logger rather than use their [default one](https://github.com/gruntwork-io/terratest/blob/90e471fe491ff724bcb61c89e38bf6cde76e5eca/modules/logger/logger.go#L17-L19). These same methods in the terratest library would use the terratest logger (which is printing logs in verbose format regardless of whether you run test with or without `-v` flag). 

That PR you mentioned enabled us to pass our own logger, whereas previously, they would just use the terratest logger and not let you modify it. Prior to this PR, we were using `logger.TestingT` logger, but it doesn't have timestamps.

Hope this makes sense!

Reply:
@ishustava Thanks, that makes sense! Would that mean in a future cleanup we could potentially remove our reimplementation and just pass in the new logger you wrote?

Reply:
I think we'd need to modify the implementation in the terratest library to be able to not re-implement these. The reason for that is that the place where you can modify the logger is when you create the Command object in the `RunKubectlAndGetOutputWithLoggerE` [here](https://github.com/hashicorp/consul-helm/blob/c4a699cb8080a574d5db07a69c96cff8a3e40989/test/acceptance/framework/k8s/kubectl.go#L41).